src/BankDataConverter.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend;
+
+use WMDE\Fundraising\Frontend\Domain\BankData;
+use WMDE\Fundraising\Frontend\Domain\Iban;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Christoph Fischer <christoph.fischer@wikimedia.de >
```

Comment:
![ocd](https://cloud.githubusercontent.com/assets/146040/12273767/f67c4396-b967-11e5-8b50-fc829e8a84ed.jpg)

Missing space after `<`!

OCD kitteh is going to erase @WMDE-Fisch from reality for greater consistency.